contrib/lib/src/space_helmet/policy.rs

```diff
@@ -0,0 +1,187 @@
+use std::borrow::Cow;
+use rocket::http::uri::Uri;
+use rocket::http::Header;
+
+
+/// [Referrer-Policy](
+/// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)
+/// description.
+pub enum ReferrerPolicy {
+    NoReferrer,
+    NoReferrerWhenDowngrade,
+    Origin,
+    OriginWhenCrossOrigin,
+    SameOrigin,
+    StrictOrigin,
+    StrictOriginWhenCrossOrigin,
+    UnsafeUrl,
+}
+
+impl Default for ReferrerPolicy {
+    fn default() -> ReferrerPolicy {
+        ReferrerPolicy::NoReferrer
+    }
+}
+
+impl<'a, 'b> From<&'a ReferrerPolicy> for Header<'b> {
+    fn from(policy: &ReferrerPolicy) -> Header<'b> {
+        let policy_string: Cow<'static, str> = match policy {
+            ReferrerPolicy::NoReferrer => "no-referrer".into(),
+            ReferrerPolicy::NoReferrerWhenDowngrade => "no-referrer-when-downgrade".into(),
+            ReferrerPolicy::Origin => "origin".into(),
+            ReferrerPolicy::OriginWhenCrossOrigin => "origin-when-cross-origin".into(),
+            ReferrerPolicy::SameOrigin => "same-origin".into(),
+            ReferrerPolicy::StrictOrigin => "strict-origin".into(),
+            ReferrerPolicy::StrictOriginWhenCrossOrigin => {
+                "strict-origin-when-cross-origin".into()
+            }
+            ReferrerPolicy::UnsafeUrl => "unsafe-url".into(),
+        };
+
+        Header::new("Referrer-Policy", policy_string)
+    }
+}
+
+pub type MaxAge = u32;
+
+/// 30 days in seconds, see the Expect-CT IETF
+/// [draft]( https://tools.ietf.org/html/draft-ietf-httpbis-expect-ct-03#page-15)
+/// for more.
+pub const EXPECTCT_MAX_AGE_DEFAULT: MaxAge = 2592000;
```

Comment:
This should be an associated constant of `ExpectCTPolicy`: `ExpectCTPolicy::MAX_AGE_DEFAULT`.